src/cpu/testers/memtest/memtest.cc

```diff
@@ -151,7 +151,7 @@ MemTest::completeRequest(PacketPtr pkt, bool functional)
     const uint8_t *pkt_data = pkt->getConstPtr<uint8_t>();
 
     if (pkt->isError()) {
-        if (!functional || !suppressFuncErrors)
+        if (!functional || !pkt->suppressFuncError())
```

Comment:
Should we keep the global parameter, too?

```suggestion
        if (!functional || !(pkt->suppressFuncError() || suppressFuncErrors))
```

Reply:
Alternatively, we can remove the `suppress_func_errors` option from `MemTest.py`

Reply:
Yes,

but I would simplify with:

```
if (!functional || !pkt->suppressFuncError() || !suppressFuncErrors) 
```

Reply:
Lol, I guess the coffee hadn't kicked in. I looked at that logic for at least 90 seconds before I decided my way made more sense.

Reply:
Ok. I've changed this to `if (!functional || !pkt->suppressFuncError() || !suppressFuncErrors)`